zookeeper-server/src/main/java/org/apache/zookeeper/common/ZKConfig.java

```diff
@@ -18,6 +18,12 @@
 
 package org.apache.zookeeper.common;
 
+import org.apache.zookeeper.Environment;
+import org.apache.zookeeper.server.quorum.QuorumPeerConfig.ConfigException;
+import org.apache.zookeeper.server.util.VerifyingFileFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
nit: revert this import reorder

Reply:
Will disable automatic organising of imports and revert the order 👍